index.bs

```diff
@@ -1762,8 +1762,8 @@ When this operation is invoked, the authenticator must perform the following pro
 1. Check if at least one of the specified combinations of {{PublicKeyCredentialType}} and cryptographic parameters in
     |credTypesAndPubKeyAlgs| is supported.
     If not, return an error code equivalent to "{{NotSupportedError}}" and terminate the operation.
-1. Check if a credential matching an [=list/item=] of |excludeCredentialDescriptorList| is present on this authenticator. If
-    so, return an error code equivalent to "{{NotAllowedError}}" and terminate the operation.
+1. Check if a credential matching an [=list/item=] of |excludeCredentialDescriptorList| is present on this authenticator for
+    the supplied |rpId|. If so, return an error code equivalent to "{{NotAllowedError}}" and terminate the operation.
```

Comment:
actually, the discrete |rpId| parameter is disappearing in PR #498 because it is duplicated in `rpEntity.id`. That fix is in PR #498.  I'd also word this a bit differently.

Reply:
Wel, I am not a native English speaker :),

Reply:
No worries -- is there a submitted issue corresponding to this PR ?  if not please submit one, and I can/will address in pr #498

Reply:
Ok, I submitted issue #654 for this.

Reply:
sigh -- @jovasco had already submitted issue #650 which I'd missed. thx.

Reply:
It has to be `rpEntity.id`. Apart from that it looks fine to me.

Reply:
@akshayku Fixed.